.github/workflows/deploy.yaml

```diff
@@ -1,7 +1,7 @@
 name: HOPR Deploy
 on:
   push:
-    branches: ['master', 'release/**', 'avado']
+    branches: ['master', 'release/**', 'avado', '!debug/**']
```

Comment:
this is a no-op

Reply:
This is meant to be used to speed up developing workflow, not meant to be a final change, see WIP